src/masonite/testing/MockRoute.py

```diff
@@ -379,3 +379,17 @@ def assertRedirect(self, redirect_uri):
         self.route.get_response()
         assert response.is_status(302) or response.is_status(301)
         assert request.redirect_url == redirect_uri
+
+    def assertDatabaseHas(self, schema, value):
```

Comment:
These should really go in the TestCase class and not the MockRoute class

Reply:
Actually that part is already in TestCase class but the above solution is for different issue. 

**Case 1**
```python
self.post("/your-api-endpoint", {
            "name": "Awesome",
}).assertDatabaseHas("table_name.column_name", "Awesome")
```

**Case 2**
```python
self.post("/your-api-endpoint", {
            "name": "Awesome",
})

self.assertDatabaseHas("table_name.column_name", "Awesome")
```

In above samples, the **Case 2** works perfectly fine because `assertDatabaseHas` already exists in TestCase, but if you try **Case 1** it gives error saying `assertDatabase is not available in MockRoute`. That's why I added those.

Reply:
@josephmancuso any reviews on this?